composer.json

```diff
@@ -30,13 +30,15 @@
         "doctrine/coding-standard": "^1.0",
         "phpunit/phpunit": "^6.5",
         "squizlabs/php_codesniffer": "^3.2",
+        "symfony/phpunit-bridge": "^4.0",
         "symfony/yaml": "~3.4|~4.0"
     },
     "suggest": {
         "symfony/yaml": "If you want to use YAML Metadata Mapping Driver"
     },
     "autoload": {
-        "psr-4": { "Doctrine\\ORM\\": "lib/Doctrine/ORM" }
+        "psr-4": { "Doctrine\\ORM\\": "lib/Doctrine/ORM" },
+        "files": ["deprecating_autoloader.php"]
```

Comment:
the commit introducing this line and the corresponding logic should be removed: it couples Doctrine to Composer, making it not-PSR4 compliant anymore. Furthermore, adding a dedicated autoloaded for Doctrine's deprecation just doens't scale. Imagine if all libs were to do so? No way on my side.